src/components/Categories/style.scss

```diff
@@ -0,0 +1,138 @@
+.movie-list__categories {
+  max-width: 100%;
+  padding: 30px 30px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #e4e4e4;
+
+  & > div {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid #acacac;
+
+    & > div {
+      display: flex;
+      width: 90px;
+      justify-content: space-between;
+    }
+  }
+
+  ul {
+    display: flex;
```

Comment:
a lot of cases you are using 
`display: flex;
justify-content: space-between;
` etc

so we can implement mixins and then reuse 
https://sass-lang.com/documentation/at-rules/mixin